jenkins-inbound-agent/Dockerfile

```diff
@@ -96,7 +96,8 @@ RUN set -eux; \
 
     #### install ansible
     pip3 install --no-cache-dir ansible==${ANSIBLE_VERSION}; \
-
+    pip3 install --no-cache-dir hvac; \
```

Comment:
Can you pin this to a specific version the same as we do for ansible already?

Reply:
pinned to latest